Dockerfile

```
@@ -1,6 +1,4 @@
-FROM python:3.8-slim-buster as python
-
-FROM python
+FROM python:3.12-bullseye
```

We probably want this to be a `slim` package. In DUNE we used `python:3.12-slim-bookworm`
```suggestion
FROM python:3.12-slim-bookworm
```